### PR TITLE
Update QA protocol to use release branches for QA

### PR DESCRIPTION
**Pull request by onebree**
Discussed during Retro on 2017-12-07: https://trello.com/c/v8sOkH5f

QA branches are effectively the same as release branches, except are children of master. We can conduct QA off of release branches, as they should be rebased against master before QA begins anyway. If changes need to be made, they can get pushed to the release branch, instead of QA, too.

### CODE REVIEW

File: protocol/development-workflow/README.md
Comment (samusgray) [inline protocol/development-workflow/README.md]: Should we add that the developer should comment on the PT story?

"Close the QA pull request on GitHub without merging and notify the feature's engineer by posting a comment on the PT story, including reasons why the story did not pass QA."

Or something like that?

**Review by samusgray — APPROVED** [review]
One question / idea and LGTM! Feel free to update or not with your own discretion and to merge without need for a second review. Cheers!

File: protocol/development-workflow/README.md
Comment (onebree) [reply protocol/development-workflow/README.md]: Yes that should be in this guide. I'll add it today, and see if any other "comment on PT" stuff should be added too. Thank you!

**Pull request merged**
